lib/screenplay/pa_messages/pa_message.ex

```diff
@@ -42,4 +42,8 @@ defmodule Screenplay.PaMessages.PaMessage do
             ((is_nil(m.end_time) and m.alert_id in ^alert_ids) or m.end_time >= ^now)
     )
   end
+
+  def get_all_messages do
+    Repo.all(from(m in __MODULE__, order_by: [desc: m.inserted_at]))
+  end
```

Comment:
**question:** How many rows do we expect this table to contain? Are we concerned about it growing over time? I ask because we aren't limiting this query in any way. Should we add pagination to this resource?

Reply:
Ah good question. I had this thought in the back of my mind but figured we could circle back to adding pagination later if we felt the need, which I'm sure we eventually would.

I'm not familiar with the specific of how often OIOs create new PAs but my guess would be that it's a little sporadic. Regardless, it seems like this table is meant to show _all_ PA Messages past, present, and future, so I guess that means this table would grow indefinitely. Unless there's a deletion workflow that I missed somewhere, or we decided to expire them from the table after some period of time.

Reply:
If it's helpful at all, some rough back of the napkin math has me guesstimating an average of <5 PA Messages created per day.